### PR TITLE
Use get and set in regression classes

### PR DESCRIPTION
**Pull request by abhinavrai44**

### CODE REVIEW

**Comment by abhinavrai44** [comment]
@lisitsyn Plz check .

File: src/shogun/regression/LeastAngleRegression.cpp [hunk 1]
```diff
@@ -185,7 +185,8 @@ bool CLeastAngleRegression::train_machine_templated(CDenseFeatures<ST> * data)
 	// main loop
 	//========================================
 	int32_t nloop=0;
-	while (m_num_active < max_active_allowed && max_corr/n_vec > m_epsilon && !stop_cond)
+	float64_t m_epsilon_value = get_epsilon();
```
Comment (karlnapf) [inline src/shogun/regression/LeastAngleRegression.cpp]: no m prefix pls, this is not a member

File: src/shogun/regression/LeastAngleRegression.cpp [hunk 2]
```diff
@@ -298,17 +299,20 @@ bool CLeastAngleRegression::train_machine_templated(CDenseFeatures<ST> * data)
 		// update estimator
 		for (index_t i=0; i < m_num_active; ++i)
 			beta[m_active_set[i]] += gamma * wA(i);
+
+		float64_t m_max_l1_norm_value = get_max_l1_norm();
```
Comment (karlnapf) [inline src/shogun/regression/LeastAngleRegression.cpp]: no prefix

Comment (karlnapf) [inline src/shogun/regression/LeastAngleRegression.cpp]: @lisitsyn: this is nuts! Storing the variables locally, obscures the code!!!

Comment (lisitsyn) [inline src/shogun/regression/LeastAngleRegression.cpp]: hey, sure, we don't need that variable at all. Just call the getter in-place.

**Comment by lisitsyn** [comment]
It looks good now. Travis complained but it was compiler crash so I restarted the job.

**Pull request merged**